covid-vue-app/test/README.md

```diff
@@ -0,0 +1,9 @@
+## How this app is tested
+
+This application uses Vue.js and Node.js. Testing is done with Jest and Selenium Webdriver. Node.js is tested with Jest, and Vue.js components are tested at ```localhost:8080``` automatically. 
+
+### server.test.js
```

Comment:
Maybe format it a little, and list out the tests that was ran in each test?

- Test if `Title` render
- Test if `Cases` is correct